src/workers/parser/pausePoints.js

```diff
@@ -124,7 +124,7 @@ function onEnter(node: BabelNode, ancestors: SimplePath[], state) {
   }
 
   if (isControlFlow(node)) {
-    addPoint(state, startLocation, isForStatement(node));
+    addStopPoint(state, startLocation);
 
     const test = node.test || node.discriminant;
     if (test) {
```

Comment:
i'd like to comment why we have this test for iteration with `while` and for loops